framework/build/enyo-build.js

```diff
@@ -1881,7 +1881,22 @@ return this._animation && this._animation.animating;
 });
 
 // dom/Dispatcher.js
-
+enyo.passiveSupported = function() {
+ passiveSupported = false;
+ try {
+  var options = {
+  get passive() {
+   passiveSupported = true;
+   return false;
+  },
+ };
+ window.addEventListener("test", null, options);
+ window.removeEventListener("test", null, options);
+ } catch (err) {
+  passiveSupported = false;
+ }
+ return passiveSupported;
+};
```

Comment:
mmh do we need this level of complexity ? We'll always be using a recent chromium-based browser on LuneOS, so I can't see the result of this test ever changing

Reply:
LuneOS isn't the only consumer of Enyo1.0 -- but this is the only fork being maintained!
I'm using it in prod for web apps running on PCs and mobile devices, as its still a decent cross-browser framework. Definitely open to a simpler way to do feature detection -- this one was [recommended by Mozilla here](https://developer.mozilla.org/en-US/docs/Web/API/EventTarget/addEventListener). The work-around before this patch was much more complex and involved measuring timing between clicks and rejecting second clicks that came in too fast.
From an app developer's perspective, this is downright elegant :1st_place_medal:

Reply:
Oh, right, I forgot the other usecases